src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5123,4 +5123,13 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_TypeReserved" xml:space="preserve">
     <value>The type name '{0}' is reserved to be used by the compiler.</value>
   </data>
+  <data name="ERR_FieldsInRoStruct" xml:space="preserve">
+    <value>Auto-implemented instance properties in readonly structs must be readonly.</value>
+  </data>
+  <data name="ERR_AutoPropsInRoStruct" xml:space="preserve">
+    <value>Instance fields of readonly structs must be readonly.</value>
```

Comment:
hmm. Thanks!!

---
In reply to: [121529802](https://github.com/dotnet/roslyn/pull/20142#discussion_r121529802) [](ancestors = 121529802)